### PR TITLE
Jane update

### PR DESCRIPTION
**Pull request by janerkim**

### CODE REVIEW

File: src/main/resources/application.properties
Comment (m-baqir) [inline src/main/resources/application.properties:1]: we want to avoid updating the properties file. if you can please remove this change. thanks

**Review by m-baqir — CHANGES_REQUESTED** [review]
please remove changes from the properties file.

File: src/main/resources/application.properties
Comment (janerkim) [reply src/main/resources/application.properties:1]: my bad forgot to omit this

**Review by m-baqir — APPROVED** [review]
looks good

**Pull request merged**